build_artifacts/v1/v1.9/v1.9.1/gpu.env.in

```diff
@@ -0,0 +1,54 @@
+# This file is auto-generated.
+conda-forge::langchain-aws[version='>=0.1.6,<0.2.0']
+conda-forge::jupyter-collaboration[version='>=1.1.0,<1.2.0']
+conda-forge::sagemaker-code-editor[version='>=1.1.0,<1.2.0']
+conda-forge::amazon-sagemaker-jupyter-ai-q-developer[version='>=1.0.4,<1.1.0']
+conda-forge::amazon-q-developer-jupyterlab-ext[version='>=3.1.0,<3.2.0']
+conda-forge::amazon_sagemaker_sql_editor[version='>=0.1.9,<0.2.0']
+conda-forge::amazon-sagemaker-sql-magic[version='>=0.1.3,<0.2.0']
+conda-forge::langchain[version='>=0.1.9,<0.2.0']
+conda-forge::fastapi[version='>=0.110.3,<0.111.0']
+conda-forge::uvicorn[version='>=0.30.1,<0.31.0']
+conda-forge::pytorch-gpu[version='>=2.0.0,<2.1.0']
+conda-forge::tensorflow-gpu[version='>=2.14.0,<2.15.0']
```

Comment:
Is this introduced recently?

Reply:
Yeah in 1.9.0 we switched to fix the gpu issue in 1.8. From 2.0 we switch back to standard tensorflow